tests/integration/rules/cpf.phpt

```diff
@@ -0,0 +1,49 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Exceptions\AllOfException;
+use Respect\Validation\Exceptions\CpfException;
+use Respect\Validation\Validator as v;
+
+try {
+    v::cpf()->check('this thing');
+} catch (CpfException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::cpf()->check('68786265440');
```

Comment:
The goal of the integration tests is to just test how the exceptions behave. That said we only need to test `check()`, `assert()`, `not(check())`, and `not(assert()` when they throw exception.

Could you remove some extra tests and put them to the unit test instead if they are not there already?